# Ejercicio 09: Análisis Exploratorio de Datos con Python
Para este ejercicio, vamos a practicar el uso de la librería Pandas
Instrucciones
Corre las siguientes celda para leer el archivo _anscombe.json_ . Los archivos json tienen un formato distinto a los csv, pero pandas los puede leer de una forma muy parecida para trabajar con ellos. 
A continuación, resuelve las siguientes tareas:

In [1]:
import pandas as pd
import numpy as np

In [2]:
df = pd.read_json("sample_data/anscombe.json")
df.head()

Series   X     Y
0      I  10  8.04
1      I   8  6.95
2      I  13  7.58
3      I   9  8.81
4      I  11  8.33

### 1. Responde como comentario: ¿Por qué fue necesario escribir *sample_data* al leer el archivo _anscombe.json_? 
(Tip: da clic en el folder que aparece en la barra de la izquierda en _Google Colab_) 

### 2. Usa la función adecuada de NumPy para calcular la **multiplicación de matrices** entre `df["X"]` y `df["Y"]`.




### 3. Crea una nueva columna en df, llamada Z. Llénala con los resultados de dividir `df["Y"]` entre `df["X"]`

### 4. Imprime las dimensiones (tamaño) de _df_

### 5. ¿Ves ese 7.58 en la columna de Y? Usa `df.iloc` para filtrar solo ese valor.

Tip: si no lo has estado ocupando, usa `df.head()`para ver cómo se ve tu tabla hasta el momento.

### 6. Calcula, para cada hilera, la mediana de "X", "Y" y "Z".

### 7. A continuación, te regalo un código. Básate en él para filtrar según los cuatro valores distintos de "Series" y guárdalo en Data Frames con el nombre de _uno_, _dos_, _tres_ y _cuatro_, manteniendo sólo las columnas X y Y.
Tip: si lo necesitas, ocupa `df["Series"].unique()` para ver qué valores tendrás que ocupar.

In [ ]:
df.loc[ df["Series"]=="I" , ["X", "Y"]]

X      Y
0   10   8.04
1    8   6.95
2   13   7.58
3    9   8.81
4   11   8.33
5   14   9.96
6    6   7.24
7    4   4.26
8   12  10.84
9    7   4.81
10   5   5.68

### 8. Calcula la media de las dos columnas de _uno_. También, la media de las dos columnas de _dos_.

### 9. Calcula la varianza de las dos columnas de _tres_. También, la varianza de las dos columnas de _cuatro_.

### 10. Calcula la correlación entre las dos variables de _dos_. También, la correlación entre las dos variables de _cuatro_.

Si lo hiciste bien, los resultados te deben dar iguales o casi iguales para los dos Dataframes. Pero, ¡los datos son distintos! (Puedes ver las cuatro tablas para confirmar cómo son distitnos).

Sacar solamente métricas para describir un problema puede llegar a ser engañoso. A veces, es necesario *visualizar* los problemas desde ángulo...